controllers/manualscalertrait_controller.go

```diff
@@ -173,11 +171,5 @@ func extractResources(workload unstructured.Unstructured) ([]cpv1alpha1.TypedRef
 func (r *ManualScalerTraitReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&oamv1alpha2.ManualScalerTrait{}).
-		Watches(&source.Kind{
```

Comment:
What happened when the deployment's replica field is changed behind this controller's back?

Reply:
https://crossplane.slack.com/archives/CTHADJCEN/p1589861506264600

Cross-linking some context here for anyone who sees this in future. In short, we're merging this functionality regression (removing a watch on deployments) because it seems to work around another, more impactful issue in that we're seeing super frequent reconciles.